kernel/src/modelingTools/NonSmoothDynamicalSystem.hpp

```diff
@@ -111,11 +111,7 @@ class NonSmoothDynamicalSystem
   /** add a dynamical system into the DS graph (as a vertex)
    * \param ds a pointer to the system to add
    */
-  inline void insertDynamicalSystem(SP::DynamicalSystem ds)
-  {
-    _topology->insertDynamicalSystem(ds);
-    _mIsLinear = ((ds)->isLinear() && _mIsLinear);
-  };
+  void insertDynamicalSystem(SP::DynamicalSystem ds);
```

Comment:
I'll just revert insertDynamicalSystem to how it was for now.  I don't think it will allow removing an include.